dev/com.ibm.ws.jdbc_fat_derby/test-applications/jdbcapp/src/jdbc/fat/derby/web/JDBCDerbyServlet.java

```diff
@@ -207,9 +212,11 @@ public void testTNTransationEnlistment() throws Throwable {
             if (!"Olmsted".equals(value))
                 throw new Exception("Incorrect value: " + value);
         } finally {
-            result.close();
-            dropTable(con, CITYTABLE);
-            con.close();
+            if (con != null) {
+                result.close();
+                dropTable(con, CITYTABLE);
+                con = null;
+	    }
```

Comment:
Is the change from `con.close()` to `con = null` deliberate here?

Reply:
Yes that is wrong.  I fixed it.